src/dotnet-grpc/Commands/CommandBase.cs

```diff
@@ -70,6 +72,11 @@ internal CommandBase(IConsole console, Project project, HttpClient httpClient)
 
         internal IConsole Console { get; set; }
         internal Project Project { get; set; }
+        private string UsingWebSdk => Project.AllEvaluatedProperties
```

Comment:
Change to method that returns bool: IsUsingWebSdk

Reply:
Yea I flip flopped on this a few times. It's odd that you'd need to convert the string to a bool back to a string here: https://github.com/grpc/grpc-dotnet/pull/443/files#diff-d2f510a9a2d14310feef2ce1dd1284e8R111. But bool looks cleaner by a little.

Reply:
Also, why a method and not a property?

Reply:
If a property is heavy then a method indicates to callers that if you need to use it multiple times then the result should be set to a local variable

A property that uses LINQ feels heavy to me.

It's not that important.